index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="logo-menubars">
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-btns">
+        <button type="button" class="menu-bars">
+          <i class="fas fa-bars"></i>
+        </button>
+        <button type="button" class="close-menubars">
+          <i class="fas fa-times"></i>
+        </button>
+      </div>
+    </div>
+    <nav class="desktop-nav">
+      <ul>
+        <li><a class="nav-link" href="#projects">Portoflio</a></li>
+        <li><a class="nav-link" href="#about-me">About</a></li>
+        <li>
+          <a class="nav-link" href="#contact">Contact <i class="far fa-envelope"></i></a>
```

Comment:
When the user clicks (or taps) any of the mobile menu options, the mobile menu should disappear, actually, it only disappears by clicking (or tapping) the close button. This could help to have a better friendly experience with the user. [Check the documentation here!](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) if you want to have additional information.
- Additional help: Right now you've created the function to close the mobile menu, just add it when the user clicks (or tap) the options inside it. 🦮